### PR TITLE
Mark a message as read before deleting it (Issue #591)

### PR DESCRIPTION
**Pull request by AurelCharp**
Hi,
I made a solution for the issue #591. Now, when we want to delete a message without reading it, the message is not marked as unread in the trash folder. 
But, in the issue, it asked to add an option for this solution and I don't know how to add an option in the settings of the app.
So, I create a first pull request with my solution and, if someone can help me to add it in option, I will changed it.

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java [hunk 1]
```diff
@@ -2810,7 +2810,7 @@ public void run() {
 
                     LocalMessage message = localFolder.getMessage(uid);
                     if (message == null
-                    || message.getId() == 0) {
+                            || message.getId() == 0) {
```
Comment (cketti) [inline k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java]: Please don't make unrelated changes or let your tools make unrelated changes.

**Comment by cketti** [comment]
Have a look through the list of merged pull requests to find one that added a feature that can be toggled via an app setting.

**Comment by AurelCharp** [comment]
Hi, 
I added a new setting in GlobalSettings/Display.
I added two new lines in string.xml in english and french but not the other languages because I don't speak others languages.
I hope I did everything the way you want.

**Comment by philipwhiuk** [comment]
Few things

1. Your PR contains all the changes from a merge you've done. This is partially because GitHub is stupid at working out what a PR should contain and what commit the PR is based off. but it makes it impossible to review. If it wasn't for point 2, you'd need to resubmit it as a new PR anyway. Generally, don't bother merging in later changes, unless they'll conflict - in which case do a new PR.
2. See #1203 for an alternative PR on this fix.
